integration/Makefile

```diff
@@ -17,3 +17,8 @@ echo_test:
 
 deployment_test:
 	./deployment_test.sh
+
+client_sdk_test:
+	cd client_sdk/go/auction && \
```

Comment:
If you create this deep dir structure, wouldn't then for consistency reasons the old scripts have to move to `client_sdk/cli` or alike?

Reply:
let's keep the CLI tests where they are, at least for now. I moved to go tests now to `client_sdk/go` and the corresponding shell script to `integration` as this also uses CLI.